src/server/lib/cookies.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from 'express';
+import { getConfiguration } from '@/server/lib/getConfiguration';
+import { logger } from './logger';
+
+interface IdapiCookie {
+  key: string;
+  value: string;
+  sessionCookie?: boolean;
+}
+
+interface IdapiCookies {
+  values: Array<IdapiCookie>;
+  expiresAt: string;
+}
+
+const { baseUri } = getConfiguration();
+
+export const setIDAPICookies = (res: Response, cookies: IdapiCookies) => {
```

Comment:
Personally, I generally prefer one export per file - I find it easier to search and read code like this (mainly because it works better with VSCode, I can CMD+P and go straight to the definition) but also because having the scope of that one function in one file makes it easier to reason about.

I totally get that this is subjective though and that we're straying close to tab vs. spaces territory here!

Reply:
Ah interesting.
I'm a big fan of CTRL/CMD+T and searching method names over using CMD+P to find filenames, or CTRL/CMD+Click on a method to go to it's definition.

Reply:
Fairs

Reply:
I've actually just refactored this into it's own file in this commit: https://github.com/guardian/gateway/pull/791/commits/d6f7e7859fbfa9b0716f41db6db9a40e3f6e37f3

After a discussion just now with @ob6160 who's going to be sending an email to gateway using the `PLAY_SESSION_2` cookie, it made sense to have a single method in that new file which first checked the `GU_email` cookie for the email, followed by the `PLAY_SESSION_2` cookie in a case where the user was coming from identity-frontend, e.g. from the registration page to the email sent page.